docker-compose.yml

```diff
@@ -20,6 +20,24 @@ services:
       - POSTGRES_USER=${DATABASE_USER}
       - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
       - POSTGRES_DB=${DATABASE_NAME}
+  redis:
+    container_name: cache
+    image: redis
+    ports:
+      - 6379:6379
```

Comment:
since you have a Redis host and port defined in your env example you could use that in this config file. 
(not necessary)
https://docs.docker.com/compose/environment-variables/set-environment-variables/